modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -88,22 +112,30 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
       _ <- SeqActionF.apply(Log.debug("Completed GHOST configuration"))
     } yield ()
 
+  // We use a dummy observation for now, since at this point, we cannot actually observe using the instrument.
   def observe(fileId: ImageFileId, expTime: Time): SeqActionF[F, ImageFileId] =
-    EitherT(ghostClient.observe(fileId, expTime.toMilliseconds.milliseconds).map(_ => fileId).attempt)
-      .leftMap {
-        case CommandResultException(_, "Message cannot be null") => Execution("Unhandled observe command")
-        case CommandResultException(_, m)                        => Execution(m)
-        case f                                                   => SeqexecException(f)
-      }
+    SeqActionF.apply((fileId, expTime)._1) // suppress unused error
+  //    EitherT(ghostClient.observe(fileId, expTime.toMilliseconds.milliseconds).map(_ => fileId).attempt)
+  //      .leftMap {
+  //        case CommandResultException(_, "Message cannot be null") => Execution("Unhandled observe command")
+  //        case CommandResultException(_, m)                        => Execution(m)
+  //        case f                                                   => SeqexecException(f)
+  //      }
 
   def endObserve: SeqActionF[F, Unit] = SeqActionF.void
 }
 
 object GHOSTController {
-  sealed abstract class BundleConfig(val configName: String)
+  sealed abstract class BundleConfig(val configName: String) {
+    def determineType(t: IFUTargetType): BundleConfig = t match {
+      case IFUTargetType.SkyPosition => BundleConfig.Sky
+      case _                         => this
+    }
+  }
   object BundleConfig {
-    case object Standard extends BundleConfig(configName = "lo")
-    case object HighRes extends BundleConfig(configName = "hi")
+    case object Standard extends BundleConfig(configName = "IFU_LORES")
```

Comment:
Where do those strings come from? Could they be read from ocs2 code?

Reply:
No, none of these are in OCS2 code. I just learned about the discovery of these strings last Wednesday. (The documentation I was reading said that these params were actually `int`s, but it turns out they're defined in the instrument software and convert there to `int`s, but you have to pass in these actual strings.

Reply:
👍